CHANGELOG.md

```diff
@@ -1 +1,11 @@
 # Changelog
+
+## 0.11.0 (TBD)
+
+### Changes
+
+- Integrated miden-client (#11).
+
+## 0.10.0 (2025-07-10)
+
+For previous changes see https://github.com/0xMiden/miden-node/blob/main/CHANGELOG.md
```

Comment:
I'm not sure we want to reference the previous log (cc @bobbinth for an opinion). But if we do, probably no need to reference `main` because it will eventually show all the other newer versions which did not host the faucet. So maybe you can reference a specific commit instead.